addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1203,10 +1203,10 @@ def execute_command_help(self, **kwargs):
         self._send_transient_message(partner, msg)
 
     def _execute_command_help_message_extra(self):
-        msg = _("""<br><br>
-            Type <b>@username</b> to mention someone, and grab their attention.<br>
-            Type <b>#channel</b> to mention a channel.<br>
-            Type <b>/command</b> to execute a command.<br>""")
+        msg = _("<br><br>"
+            "Type <b>@username</b> to mention someone, and grab their attention.<br>"
+            "Type <b>#channel</b> to mention a channel.<br>"
+            "Type <b>/command</b> to execute a command.<br>")
```

Comment:
Doesn't this break translatability?

Reply:
why? I think the difference will be removing the `/n`s and spacing.

Reply:
Was the reasoning of this change to ease deep equal of `assertBus`?

Not gonna lie, I understand the desire to add tests to increase confidence on the code, but also putting so much important on testing can lead to functional mistakes, dissuade making functionally meaningful changes to the code, etc.

Yes, I take a bit of this opportunity to spit on over-testing the code, because we are rational enough on the drawbacks of (bad) tests. Sure we could say "don't make bad tests then"... But if we force any changes in the code to always make a test, then you motivate developers to either make bad tests or just not make a PR to change the code.

Reply:
This change is mainly for the `assertBus` yes. Since the spacing and `\n` are better to be checked if both are removed.

Reply:
For me, I think it is fine to add a test to a fix so that it does not break easily, only if we don't make a lot of changes to the function code. In this case, I think it is fine. The readability is not influenced a lot and functionally it is working ( asked the i18 team ).